[2024-11-28T15:20:01.175+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-28T15:20:01.188+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_airflow_dag.get_datetime scheduled__2024-11-28T15:19:00+00:00 [queued]>
[2024-11-28T15:20:01.194+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_airflow_dag.get_datetime scheduled__2024-11-28T15:19:00+00:00 [queued]>
[2024-11-28T15:20:01.194+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-11-28T15:20:01.205+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): get_datetime> on 2024-11-28 15:19:00+00:00
[2024-11-28T15:20:01.214+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=15917) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-28T15:20:01.215+0000] {standard_task_runner.py:72} INFO - Started process 15924 to run task
[2024-11-28T15:20:01.217+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'scheduled__2024-11-28T15:19:00+00:00', '--job-id', '33617', '--raw', '--subdir', 'DAGS_FOLDER/first_***_dag.py', '--cfg-path', '/tmp/tmpv44o7bbc']
[2024-11-28T15:20:01.218+0000] {standard_task_runner.py:105} INFO - Job 33617: Subtask get_datetime
[2024-11-28T15:20:01.254+0000] {task_command.py:467} INFO - Running <TaskInstance: first_airflow_dag.get_datetime scheduled__2024-11-28T15:19:00+00:00 [running]> on host 5365fe4338c6
[2024-11-28T15:20:01.311+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='first_***_dag' AIRFLOW_CTX_TASK_ID='get_datetime' AIRFLOW_CTX_EXECUTION_DATE='2024-11-28T15:19:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-28T15:19:00+00:00'
[2024-11-28T15:20:01.312+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-28T15:20:01.312+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-28T15:20:01.313+0000] {logging_mixin.py:190} INFO - Current task name:get_datetime state:running start_date:2024-11-28 15:20:01.188544+00:00
[2024-11-28T15:20:01.313+0000] {logging_mixin.py:190} INFO - Dag name:first_***_dag and current dag run status:running
[2024-11-28T15:20:01.313+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-28T15:20:01.313+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-28T15:20:01.314+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'date']
[2024-11-28T15:20:01.323+0000] {subprocess.py:86} INFO - Output:
[2024-11-28T15:20:01.326+0000] {subprocess.py:93} INFO - Thu Nov 28 15:20:01 UTC 2024
[2024-11-28T15:20:01.326+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-28T15:20:01.342+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-28T15:20:01.343+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, run_id=scheduled__2024-11-28T15:19:00+00:00, execution_date=20241128T151900, start_date=20241128T152001, end_date=20241128T152001
[2024-11-28T15:20:01.349+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-28T15:20:01.350+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-28T15:20:01.350+0000] {logging_mixin.py:190} INFO - Dag name:first_***_dag queued_at:2024-11-28 15:20:00.208419+00:00
[2024-11-28T15:20:01.350+0000] {logging_mixin.py:190} INFO - Task hostname:5365fe4338c6 operator:BashOperator
[2024-11-28T15:20:01.390+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-28T15:20:01.400+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-28T15:20:01.402+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
